contrib/src/main/java/com/datatorrent/contrib/hdht/HDHTWriter.java

```diff
@@ -465,8 +466,8 @@ private void writeFileWithPurge(Bucket bucket, BucketFileMeta meta, RangeSet<Sli
     TreeMap<Slice, Slice> fileData = new TreeMap<>(keyComparator);
 
     /* Check if there is data in initial part of file before next purge range */
-    Slice key = new Slice(null, 0, 0);
-    Slice value = new Slice(null, 0, 0);
+    Slice key = new Slice(new byte[] {0}, 0, 1);
```

Comment:
@vrozov There seems to be a difference in [tag 1.3.0](https://github.com/DataTorrent/Netlet/blob/v1.3.0/src/main/java/com/datatorrent/netlet/util/Slice.java#L42) and [master](https://github.com/DataTorrent/Netlet/blob/master/src/main/java/com/datatorrent/netlet/util/Slice.java#L42) for the null check. @tushargosavi decompiled netlet 1.3.0 from the maven repo and we see that the null check is missing.